exercise2/ado77/common/sizeOf.mjs

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+var ECMA_SIZES = {
+    STRING: 2,
+    BOOLEAN: 4,
+    NUMBER: 8
+};
+// var Buffer = require('buffer').Buffer
```

Comment:
What are your thoughts about committing commenting code?

Reply:
Lol, if I'm honest, I don't use that file... I was curious about the size of my objects/arrays...